deploy/examples/multus-validation.yaml

```diff
@@ -0,0 +1,138 @@
+# allowPrivilegedContainer: true
+# allowHostDirVolumePlugin: true
+# allowHostPID: false
+# # set to true if running rook with host networking enabled
+# allowHostNetwork: true
+# # set to true if running rook with the provider as host
+# allowHostPorts: true
+# priority:
+# allowedCapabilities: ["MKNOD"]
+# allowHostIPC: true
+# readOnlyRootFilesystem: false
+# # drop all default privileges
+# requiredDropCapabilities: ["All"]
+# defaultAddCapabilities: []
+# runAsUser:
+#   type: RunAsAny
+# seLinuxContext:
+#   type: RunAsAny
+# fsGroup:
+#   type: RunAsAny
+# supplementalGroups:
+#   type: RunAsAny
+# seccompProfiles:
+#   - "*"
+# volumes:
+#   - configMap
+#   - emptyDir
+#   - projected
+# users:
+#   - system:serviceaccount:rook-ceph:rook-ceph-multus-validation # serviceaccount:namespace:cluster
+---
+# Aspects of multus validation job that require access to the operator/cluster namespace
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: rook-ceph-multus-validation
+  namespace: rook-ceph # namespace:cluster
+rules:
+  - apiGroups: [""]
+    resources: ["configmaps", "configmaps/finalizers", "pods"]
+    verbs: ["get", "list", "create", "update", "delete"]
+  - apiGroups: ["apps"]
+    resources: ["daemonsets"]
+    verbs: ["list", "create", "delete", "deletecollection"]
+  - apiGroups: ["k8s.cni.cncf.io"]
+    resources: ["network-attachment-definitions"]
+    verbs: ["get"]
+  - apiGroups: ["batch"]
+    resources: ["jobs"]
+    verbs: ["get", "list", "delete"]
+---
+# Allow the multus validation job to run in this namespace
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: rook-ceph-multus-validation
+  namespace: rook-ceph # namespace:cluster
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: rook-ceph-multus-validation
+subjects:
+  - kind: ServiceAccount
+    name: rook-ceph-multus-validation
+    namespace: rook-ceph # namespace:cluster
+---
+# Service account for job that validates multus configuration
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: rook-ceph-multus-validation
+  namespace: rook-ceph # namespace:cluster
+# imagePullSecrets:
+#   - name: my-registry-secret
+---
+# A job that runs the multus validation tool
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: rook-ceph-multus-validation
+  namespace: rook-ceph # namespace:cluster
+  labels:
+    app: rook-ceph-multus-validation
+spec:
+  template:
+    metadata:
+      labels:
+        app: rook-ceph-multus-validation
+    spec:
+      serviceAccountName: rook-ceph-multus-validation
+      containers:
+        - name: multus-validation
+          image: rook/ceph:master
+          command: ["rook"]
+          # TODO: Insert the NAD name for public network and cluster network
+          # If you want to use any other flags along with the basic command,
+          # add the `--help` flag in the end to see the list of flags available, and use accordingly.
```

Comment:
Additionally, it seems best to me to move this doc to the header of the file. It will be easiest for users if there is one primary place to read about how to use the file, and that primary doc is at the very top. Any other comments in the file should be supplementary, to help users find "where" to make changes with brief reminders.